docs/Operators.md

```diff
@@ -7630,39 +7826,79 @@ This version of the operator has been available since version 16 of the default
 #### Attributes
 
 <dl>
-<dt><tt>align_corners</tt> : int (default is 0)</dt>
-<dd>If align_corners=1, the extrema (-1 and 1) are considered as referring to the center points of the input's corner pixels. If align_corners=0, they are instead considered as referring to the corner points of the input's corner pixels, making the sampling more resolution agnostic.</dd>
+<dt><tt>align_corners</tt> : int32 (default is 0)</dt>
+<dd>This attribute must be either 0 or 1.</dd>
 <dt><tt>mode</tt> : string (default is bilinear)</dt>
-<dd>Three interpolation modes: bilinear (default), nearest and bicubic.</dd>
+<dd>This attribute must be one of the strings 'bilinear', 'nearest' and 'bicubic'.</dd>
 <dt><tt>padding_mode</tt> : string (default is zeros)</dt>
-<dd>Support padding modes for outside grid values: `zeros`(default), `border`, `reflection`. zeros: use 0 for out-of-bound grid locations, border: use border values for out-of-bound grid locations, reflection: use values at locations reflected by the border for out-of-bound grid locations. If index 0 represents the margin pixel, the reflected value at index -1 will be the same as the value at index 1. For location far away from the border, it will keep being reflected until becoming in bound. If pixel location x = -3.5 reflects by border -1 and becomes x' = 1.5, then reflects by border 1 and becomes x'' = 0.5.</dd>
+<dd>This attribute must be one of the strings 'zeros', 'border' and 'reflection'</dd>
+<dt><tt>alpha</tt> : float (default is -0.75)</dt>
+<dd>This attribute is used for bilinear and bicubic interpolation. See above for the formulas involving alpha. The default value matches that of PyTorch.
+<dt><tt>nearest_mode</tt> : string (default is round_to_even)</dt>
+<dd>This attribute specifies the rounding mode used in nearest interpolation and must be one of the strings 'round_prefer_floor' (round .5 down), 'round_prefer_ceil' (round .5 up), 'round_to_even' (round .5 to closest even integer). If the rounding mode is not nearest then this attribute is ignored, though it must still be one of the indicated strings.</dd>
 </dl>
 
+
+
 #### Inputs
 
 <dl>
-<dt><tt>X</tt> (differentiable) : T1</dt>
-<dd>4-D tensor of shape (N, C, H, W), where N is the batch size, C is the numbers of channels, H and W are the height and width of the input data.</dd>
-<dt><tt>grid</tt> (non-differentiable) : T1</dt>
-<dd>Input offset, 4-D tensor of shape (N, H_out, W_out, 2), where H_out and W_out are the height and width of grid and output, Grid specifies the sampling pixel locations normalized by the input spatial dimensions. Therefore, it should have most values in the range of [-1, 1]. If grid has values outside the range of [-1, 1], the corresponding outputs will be handled as defined by padding_mode.</dd>
+<dt><tt>input</tt> (differentiable) : T1</dt>
```

Comment:
We set this input name as `X` in kernel implementation and example. So `X` should be the input1 name.

Reply:
Yep, let me fix that.